dias/analyzers/sensitivity_analyzer.py

```diff
@@ -180,6 +180,8 @@ def run(self):
                 files, datasets=['reverse_map', 'flags/inputs'],
                 apply_gain=False, renormalize=False)
 
+            flag_ind = data.flags
```

Comment:
I might be wrong, but can't this even be
```
flag_ind = data.flags['inputs']
```
?

Not sure if it makes a difference in performance, though.

Reply:
Yeah, `flag_ind = data.flags` and then using that variable did not make difference. The time was ~30 min for 1.5 hours file.

Reply:
I would suggest trying with `['inputs']` then. If you want to profile the analyzer, have a look at this: https://docs.python.org/3/library/profile.html

Reply:
Tried `['inputs']`, does not seem to make a difference. On an average, it is taking 30-40 min for each file (that is 1.5 hours of data).